app-emulation/docker-runc/files/0001-temporarily-disable-selinux.GetEnabled-error-checks.patch

```diff
@@ -0,0 +1,58 @@
+From 7264883458e7bdcd23152495ac141a24d17b8db9 Mon Sep 17 00:00:00 2001
+Message-Id: <7264883458e7bdcd23152495ac141a24d17b8db9.1564136934.git.dongsu@kinvolk.io>
+From: Dongsu Park <dongsu@kinvolk.io>
+Date: Fri, 26 Jul 2019 12:19:37 +0200
+Subject: [PATCH] temporarily disable selinux.GetEnabled() error checks
+
+---
+ libcontainer/configs/validate/validator.go | 4 ----
+ utils_linux.go                             | 4 ----
+ 2 files changed, 8 deletions(-)
+
+diff --git a/libcontainer/configs/validate/validator.go b/libcontainer/configs/validate/validator.go
+index 3b42f301..bace067d 100644
+--- a/libcontainer/configs/validate/validator.go
++++ b/libcontainer/configs/validate/validator.go
+@@ -8,7 +8,6 @@ import (
+ 
+ 	"github.com/opencontainers/runc/libcontainer/configs"
+ 	"github.com/opencontainers/runc/libcontainer/intelrdt"
+-	selinux "github.com/opencontainers/selinux/go-selinux"
+ )
+ 
+ type Validator interface {
+@@ -99,9 +98,6 @@ func (v *ConfigValidator) security(config *configs.Config) error {
+ 		!config.Namespaces.Contains(configs.NEWNS) {
+ 		return fmt.Errorf("unable to restrict sys entries without a private MNT namespace")
+ 	}
+-	if config.ProcessLabel != "" && !selinux.GetEnabled() {
+-		return fmt.Errorf("selinux label is specified in config, but selinux is disabled or not supported")
```

Comment:
`config.ProcessLabel` hmm, maybe we could modify the configuration rather than changing the code?

Reply:
Maybe that would be just removing the `+selinux` line when building, don't know and it takes long to test such a patch. Actually it would be good to enable SELinux in Flatcar.

Reply:
By keeping the support compiled, we can try if it works when the files get the correct labels.

Reply:
About changing the config, I'm not sure how to do that, because the runtime config is generated outside of runc, in this case by docker.
So we need to somehow patch docker as well. Not so simple.

Removing selinux build tags should be also done for docker as well.

So for now I have no better idea, I just needed a simple fix. ;-)

Reply:
Thanks for additional context @dongsupark, your patch seems reasonable then, if it doesn't break other things :+1: